slices/ca-certificates.yaml

```diff
@@ -15,6 +23,29 @@ slices:
         content.read(certs_dir + path) for path in content.list(certs_dir)
       ]
       content.write("/etc/ssl/certs/ca-certificates.crt", "".join(certs))
+      names = [
+        "mozilla/{}".format(name) for name in content.list(certs_dir)
+      ]
+      content.write("/etc/ca-certificates.conf", "\n".join(names))
+
+  # The `_data` slice only provides the bundled certificates and the config file.
+  # To keep the individual certificates, this slice should be included.
+  data-with-certs:
+    essential:
+      - ca-certificates_data
+    contents:
+      /usr/share/ca-certificates/mozilla/*:
```

Comment:
Running locally `chisel cut --release ./ --root /home/ubuntu/chisel/ ca-certificates_data-with-certs` I get :
```
...
2024/07/08 13:00:07 Extracting files from package "ca-certificates"...
error: cannot extract from package "ca-certificates": when using wildcards source and target paths must match: /usr/share/ca-certificates/mozilla/*
```

Reply:
Which version `chisel` are you running? There is a known bug in v0.9.1, and v0.10.0 fixed it.

Reply:
You are right, with v0.10.0 that error disappears (unless I am wrong v0.9.1 is used in rockcraft)